src/pages/listpage/CardListPage.jsx

```diff
@@ -0,0 +1,78 @@
+import { RecipientsAPI } from 'data/CallAPI';
+import { useState, useEffect } from 'react';
+import CardList from 'components/card/CardList';
+import TextDropdownField from 'components/textfield/TextDropdownField';
+import TextInputField from 'components/textfield/TextInputField';
+import EmojiToggle from 'components/EmojiToggle';
+import "pages/listpage/CardListPage.scss";
+
+function HotSort(recipient) {
+  if (!recipient || !Array.isArray(recipient.results)) {
+    //레시피언트 없는지와 레시피언트리절트가 배열인지 확인
+    return [];
+  }
+  return [...recipient.results].sort((a,b) => {
+    if(b.messageCount === a.messageCount) {
+      return b.reactionCount - a.reactionCount;
+    }
+    return b.messageCount - a.messageCount;
+  });
+}
+
+function DateSort(recipient) {
+  if (!recipient || !Array.isArray(recipient.results)) {
+    //레시피언트 없는지와 레시피언트리절트가 배열인지 확인
+    return [];
+  }
+  return [...recipient.results].sort((a,b) => b.createdAt - a.createdAt);
+}
+
+function CardListPage() {
+  const test = ['a', 'b', 'c', 'd'];
+  const [recipient, setRecipient] = useState({});
+
+  useEffect(() => {
+    const getRecipient = async () => {
+      try {
+        const response = await RecipientsAPI('get');
+        setRecipient(response);
```

Comment:
밑에 코드처럼 limit, offset 받도록 수정해주시구 (코드는 틀릴수도 있으니 참고만 ㅎㅎ)
id가 붙으면 limit, offset이 필요하지않아요~!
```
export function RecipientsAPI(Method, id = null, body = null, limit=4, offset=1) {
  let query = `${API_KEY}/recipients`;
  if (id) {
      query = `${query}/${id}/`;
   } else {
     query = `${query}?limit=${limit}&offset=${offset}`
  }
  return Axios(Method, query, body);
}

```

```suggestion
const response = await RecipientsAPI('get', null, 4);
setRecipient(response);
```
요렇게 처음엔 4개만 받아오게 해주시면 좋을거같아요!
그다음에 화살표 눌렀을때 추가로 받아와서 리렌더링되게끔~!